### PR TITLE
Update test_list.py

### PR DESCRIPTION
**Pull request by Dmytro-Mld**
Added broken test

### CODE REVIEW

**Comment by Dmytro-Mld** [comment]
Correct code

File: test_list.py [hunk 1]
```diff
@@ -8,12 +8,18 @@ def test_roman_tkalenko_fi_13(self):
         self.assertEqual(len(['a']), 1)
         self.assertEqual(len(['a', 'b']), 2)
 
-    def test_roman_tkalenko_2(self):
-        self.assertEqual(2, 2)
+    def test_polina_kozarovitskaya(self):
```
Comment (tkroman) [inline test_list.py:11]: why is this in your pr?

File: test_list.py [hunk 2]
```diff
@@ -8,12 +8,18 @@ def test_roman_tkalenko_fi_13(self):
...
+        t = [1, 2, 3]
+        t.clear()
+        self.assertEqual(len(t), 0)
+
 
     def test_illia_kripaka_fi_94_2(self):
         self.assertEqual(2*[1, 3, 5], [1, 3, 5, 1, 3, 5])
 
+        def test_dmytro_moldovan_(self):
```
Comment (tkroman) [inline test_list.py:20]: wrong indentation

**Review by tkroman — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS [hunk 3]
```diff
@@ -1,3 +1,6 @@
 tkroman
+ikripaka
+Dmytro-Mld
 PJeys
-ikripaka
\ No newline at end of file
+ikripaka
```
Comment (tkroman) [inline CONTRIBUTORS:5]: x2 (see line 5)

File: CONTRIBUTORS [hunk 4]
```diff
@@ -1,9 +1,10 @@
 tkroman
+ikripaka
```
Comment (nogov) [inline CONTRIBUTORS:2]: PJeys mentioned twice, ikripaka was moved.
This file's diff should contain only the addition of your account name. Fix, please.

**Review by nogov — CHANGES_REQUESTED** [review]

**Review by nogov — DISMISSED** [review]

**Review by tkroman — APPROVED** [review]

**Pull request merged**